src/options/smt_options.toml

```diff
@@ -620,25 +620,40 @@ header = "options/smt_options.h"
 [[option]]
   name       = "solveBVAsInt"
   category   = "undocumented"
-  long       = "solve-bv-as-int=N"
-  type       = "uint32_t"
-  default    = "0"
+  long       = "solve-bv-as-int"
+  type       = "bool"
+  default    = "false"
   read_only  = true
   help       = "attempt to solve BV formulas by translation to integer arithmetic (experimental)"
 
 [[option]]
-  name       = "bvToIntIand"
+  name       = "solveBVAsIntMode"
```

Comment:
Actually I think this can be merged with solveBVAsInt  (OFF is another mode?)

Reply:
Fixed. Open to more merging suggestions :)